.github/workflows/docker-build-push-google.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: build and push docker image to google
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: "github actions runner"
+        required: true
+        type: string
+      workload-identity-provider:
+        description: "projects/<number>/locations/global/..."
+        required: true
+        type: string
+      service-account:
+        description: "github-artifactregistry@..."
+        required: true
+        type: string
+      registry:
+        description: "registry url"
+        required: true
+        type: string
+      gcp:
+        description: "google cloud project id"
+        required: true
+        type: string
+      repository:
+        description: "registry repo"
+        required: true
+        type: string
+      image_name:
```

Comment:
can we use imageName? or image-name, I don't like snake_case or only `image`

Reply:
In use is already - on other place, so - would be better like upper case